.github/workflows/deploy-stage-api.yml

```
@@ -41,3 +51,18 @@ jobs:
           DISTRIBUTION: E2MLRMA1VTVDHX
           PATHS: /bcd/api/*
         run: aws cloudfront create-invalidation --distribution-id "$DISTRIBUTION" --paths "$PATHS"
+
+      - name: Authenticate with GCP
+        uses: google-github-actions/auth@v0
+        with:
+          token_format: access_token
+          service_account: deploy-stage-bcd@${{ secrets.GCP_PROJECT_NAME }}.iam.gserviceaccount.com
+          workload_identity_provider: projects/${{ secrets.WIP_PROJECT_ID }}/locations/global/workloadIdentityPools/github-actions/providers/github-actions
+
+      - name: Setup gcloud
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Sync Rumba API
+        run: |-
+          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -r out/ gs://bcd-stage-mdn/main/bcd/api/
+          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -d -r out/v0/current gs://bcd-stage-mdn/main/bcd/api/v0/current
```

Let's drop the `main/bcd` prefix and sync directly into the `api` folder at the bucket root:

```suggestion
          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -r out/ gs://bcd-stage-mdn/api/
          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -d -r out/v0/current gs://bcd-stage-mdn/api/v0/current
```